bottomless/src/replicator.rs

```diff
@@ -684,6 +684,9 @@ impl Replicator {
         db_path: &Path,
         compression: CompressionKind,
     ) -> Result<ByteStream> {
+        if !tokio::fs::try_exists(db_path).await? {
+            bail!("database file was not found at `{}`", db_path.display())
+        }
```

Comment:
that's a redundant check now, isn't it? We'll get the exact same information below, that file was not found, plus its path

Reply:
I've just found the description a little bit missing. Since it's only saying `No such file or directory` while we're copying things from one file to another, it's cleaner to say which file is missing.